crates/api_common/src/lib.rs

```diff
@@ -252,12 +253,34 @@ pub fn user_show_nsfw(local_user_view: &Option<LocalUserView>) -> bool {
   }
 }
 
+/// A helper method for showing read posts
+pub fn user_show_read_posts(local_user_view: &Option<LocalUserView>) -> bool {
+  match local_user_view {
+    Some(uv) => uv.to_owned().local_user.show_read_posts,
+    None => true,
+  }
```

Comment:
This could also be a simple map.

Reply:
I think doing it that way, i'll be able to get rid of these helper functions entirely. I'll do that shortly.